tpp-app/src/home/pages/CycleScreen.js

```diff
@@ -129,37 +129,42 @@ export default function CycleScreen ({navigation}){
   const tipInvisibleStyle = {
     marginBottom: tabBarHeight
   }
-  const cardContainerStyle = showTip ? styles.cardContainer : Object.assign({}, styles.cardContainer, tipInvisibleStyle);
+  const tipVisibleStyle = {
+    marginBottom: 50
+  }
+  const cardContainerStyle = showTip ? Object.assign({}, styles.cardContainer, tipVisibleStyle) : Object.assign({}, styles.cardContainer, tipInvisibleStyle);
   return (
     <SafeAreaView style={styles.container}>
       <ImageBackground source={background} style={styles.container}>    
         {/* View that contains all the relevant cards */}
-        <ScrollView contentContainerStyle={cardContainerStyle}>
+        <ScrollView>
           {/* Period Notification (Period in X days) */}
-          {showTip && (
-          <PeriodNotification daysTillPeriod={daysTillPeriod}>
-            <Paddy style={styles.paddyIcon}/>
-          </PeriodNotification>
-          )}
-          <CycleCard 
-            periodDays={periodDays} 
-            daysSinceLastPeriod={daysSinceLastPeriod} 
-            cycleDonutPercent={cycleDonutPercent}
-            showTip={showTip}
-          />
-          <SafeAreaView style={[styles.rowContainer, styles.infoCardContainer, styles.element]}>
-            <InfoCard header="Average period length" days={avgPeriodLength} backgroundColor="#FFDBDB">
-              <BloodDrop fill="red" style={styles.icon}/>
-            </InfoCard>
-            <InfoCard header="Average cycle length" days={avgCycleLength} backgroundColor="#B9E0D8">
-              <Calendar fill="red" style={styles.icon}/>
-            </InfoCard>
-          </SafeAreaView>
-          <MinimizedHistoryCard 
-            navigation={navigation} 
-            intervals={intervals}
-            onPeriod={periodDays !=0}
-          />
+          <View style={cardContainerStyle}>
```

Comment:
use safe area view here as well unless there's a good reason not to?

Reply:
u are right and there is no good reason.